### ISSUE TITLE
Shows adds but not monitored correctly so nothing downloads

### ISSUE THREAD

**Issue by treverehrfurth** [preram48/sonarr-trakt-tv#14]
When I add a show. It does get added to Sonarr and all the seasons are toggled for "Monitored" however none of the episodes within the seasons are and therefore it says there isn't any missing episodes and will not download anything.

I have to go in, toggle the season monitored flag on and off and then it will ping the episodes as missing and search for them on the next scan.

So close yet so far... but this still requires me to go in and manually toggle every show and every season within making it kind of...useless

Hoping someone has something to fix this!
![Screenshot_2](https://user-images.githubusercontent.com/25590075/88711052-31330b00-d0dd-11ea-9bfa-c5a06452788f.png)

**Comment by treverehrfurth** [comment]
Is there a place to change it so that when a show is added it also searching for everything? like the "add and search" button

**Comment by BackedUpBooty** [comment]
Yeah I've got the same thing happening. Also for some reason all 'use season folder' settings on imported shows using this (still helpful) service are set to 'no'.
